docs/install.md

```diff
@@ -93,3 +93,32 @@ To update the News app use change into the **nextcloud/apps/news/** directory us
 
     git pull --rebase origin master
     make
+
+## Uninstall with cleanup
+
+First uninstall the app via the web-interface or via occ:
+    
+    ./occ app:remove news
+
```

Comment:
Can you use the  triple backtick format here? I've always found the whitespace based format to be very akward.
```suggestion
```console
./occ app:remove news
\`\`\`
```

Reply:
Indeed you can, I was to lazy to try.